apps/storybook/src/navigation/Breadcrumb.stories.tsx

```diff
@@ -92,41 +83,41 @@ export const WithDropdown: Story = {
             />
             <DropdownMenuContent align="start">
               <DropdownMenuItem>
```

Comment:
we can now use proper composition here
```suggestion
              <DropdownMenuItem asChild>
```

and remove the `<Text />` element keeping just the `<a />` element